### ISSUE TITLE
Python script for automated import of data from an MdH Core

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#9]
## User story
1. As a user
2. I want the MdHOS container to automatically import data from an MdH Core
3. So that i don't have to feed jsons objects into its database manually

## Acceptance criteria
* Create a simple python script that imports some data from our MdH Core into the OpenSearch database.
Just fetch some random data from the MdH. No specific filters or anything required. Limit the amount of fetched data to a reasonable number. We just need some test data to start...
It's probably easiest to use the [MdH Python SDK](https://metadatahub.de/documentation/2.3/mdh/interfaces/python.html)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by leondaniel22** [comment]
We created two branches: 

- **test-data-import**: Omar, Martin, Amir and me created a script that extracts data from the MetaDataHub
- **opensearch-import-script**: Amir, Omar and me created a script that takes test Data and adds it to our OpenSearch Node Docker container

We are planning to combine those two scripts so that we can directly add data from the MetaDataHub to our OpenSearch Node
